README.md

```diff
@@ -248,8 +248,15 @@ QAndQDotBounds waits for a biorbd model and returns a structure with the minimal
 How convenient!
 ```python
 x_bounds = QAndQDotBounds(biorbd_model)
-```
-Then, override the first and last column to be 0, that is the sideways and rotation to be null for both the position and the velocities
+
+
+The first element of x_bounds is the degree of freedom declared in order in the bio.Mod file.
+In this case, the degree of freedom with index 0 is translation y, and the index 1 refers to rotation x. 
+Finally, the index 2 and 3 are respectively the velocity of translation y and rotation x 
```

Comment:
This would make more sense after the sentence (The pendulum is required to start in a downward position (0 rad) and to finish in an upward position (3.14 rad) with no velocity at start and end nodes.)